cpp/src/copying/contiguous_split.cu

```diff
@@ -43,6 +42,7 @@
 #include <thrust/reduce.h>
 #include <thrust/scan.h>
 #include <thrust/transform.h>
+#include <thrust/transform_scan.h>
```

Comment:
I don't see this function is used anywhere?

Reply:
this and the next one handled in https://github.com/rapidsai/cudf/pull/13260/commits/8f9dbbd9accf489eb29ec8b1cd8c60419b2104eb